packages/console/src/components/AppContent/components/Sidebar/hook.tsx

```diff
@@ -111,7 +112,7 @@ export const useSidebarMenuItems = (): {
           Icon: Document,
           title: 'docs',
           externalLink: `https://docs.logto.io/${conditionalString(
-            language !== Language.English && language.toLowerCase()
+            language === Language.Chinese && language.toLowerCase()
```

Comment:
will be better to maintain a l10n set of docs

Reply:
Oh, I see. So right now the new set of docs languages will only contain English and Chinese